src/utilities/inputProcessors.ts

```diff
@@ -149,6 +150,20 @@ export const processInputReleaseBranches = (): BranchSpec[] | undefined => {
   return validateInputReleaseBranches(parsedInput);
 };
 
+export const processInputConfigFile = (): string | undefined => {
+  const file = parseFileList(getInput(InputParameters.ConfigFile)).shift();
+
+  if (file === undefined) {
+    return;
+  }
+
+  if (/\\.(?:yaml|yml)?/u.exec(file) === null) {
```

Comment:
Please cache the regex in a const. Also the regex can be optimized to make the `a` in yaml an optional character.
`/\\.(?:ya?ml)?/u`

Reply:
Sure 👍🏻 
47d227af4a0aed7ccdbdaac0ea23295cc6a359ca

Reply:
Please cache it _outside_ the function with a const name:
`MATCH_YAML_EXTENSION_REGEXP`

Reply:
I wonder tho, is there any gain to be made here?

Reply:
Considering above, I'll leave it.